pkg/config/config.go

```diff
@@ -561,27 +553,26 @@ func (c *ContainersConfig) Validate() error {
 // The parameter `onExecution` specifies if the validation should include
 // execution checks. It returns an `error` on validation failure, otherwise
 // `nil`.
-func (c *NetworkConfig) Validate(onExecution bool) error {
-	if onExecution {
-		err := IsDirectory(c.NetworkConfigDir)
+func (c *NetworkConfig) Validate() error {
```

Comment:
Are both `Validate` methods intended to be called by the consumers?

It seems strange as it would imply that we could return invalid configurations.

Reply:
The problem I was looking at was the situation where buildah was using it and did not care about networkconfig or libpodconfig.

Reply:
So the lack of a cniplugin directory is not really something we should block against.